app/services/course/assessment/question/programming/python/python_package_service.rb

```diff
@@ -198,6 +174,28 @@ def generate_zip_file(data_files_to_keep)
     tmp
   end
 
+  def zip_test_files(test_type, zip)
```

Comment:
@weiqingtoh  But that would still be 13/10 😆. What are the two types of cops that I need to disable?